### PR TITLE
migrate to tailwind css v2; update dependencies

### PR DESCRIPTION
**Pull request by thecodingwizard**
Does anyone notice any issues / inconsistencies (particularly for styles & colors)?

### CODE REVIEW

**Comment by HarryWangATX** [comment]
@thecodingwizard the psum widget needs fixing

**Comment by HarryWangATX** [comment]
^for mobile im not sure about desktop

**Comment by HarryWangATX** [comment]
image

**Comment by HarryWangATX** [comment]
seems only the first widget is broken. the second one is fine

**Comment by HarryWangATX** [comment]
image

image

**Comment by HarryWangATX** [comment]
the hint styles might have been messed up??? before i collapse it its two lines but afterward it turns into 1

File: content/3_Silver/Prefix_Sums.mdx [hunk 1]
```diff
@@ -193,7 +193,7 @@ export const problems = {
 
 <FocusProblem problem={problems.sample[0]} />
 
-## Resources
+## Resources12342
```
Comment (dolphingarlic) [inline content/3_Silver/Prefix_Sums.mdx:196]: Did you forget to revert this?

Comment (thecodingwizard) [reply content/3_Silver/Prefix_Sums.mdx:196]: yes :c good catch thanks!

**Pull request merged**